components/uaepass-authenticator/src/test/java/org/wso2/carbon/identity/uaepass/authenticator/UAEPassAuthenticatorTest.java

```diff
@@ -0,0 +1,729 @@
+    private static Map<String, String> authenticatorProperties;
+    private int TENANT_ID = 1234;
+    private Map<String, String> paramValueMap;
+    private static String accessToken = "fd2fffca-b5e2-466d-aac9-207382497b88";
+    private static String idToken = "eyJ4NXQiOiJOVGRtWmpNNFpEazNOalkwWXpjNU1tWm1PRGd3TVRFM01XWXdOREU1TVdSbFpEZzROemM0Wk" +
+            "EiLCJraWQiOiJNell4TW1Ga09HWXdNV0kwWldObU5EY3hOR1l3WW1NNFpUQTNNV0kyTkRBelpHUXpOR00wWkdSbE5qSmtPREZrWkRSaU9U" +
+            "RmtNV0ZoTXpVMlpHVmxOZ19SUzI1NiIsImFsZyI6IlJTMjU2In0.eyJhdF9oYXNoIjoiNzgwOE5OTlphN2ZfdHFzaVVzNG5vZyIsInN1Yi" +
+            "I6Ijk0NzEyMTg0NTE4IiwiZ2VuZGVyIjoiVW5rbm93biIsImFtciI6WyJVQUVQYXNzQXV0aGVudGljYXRvciIsIkJhc2ljQXV0aGVudGl" +
+            "jYXRvciJdLCJpc3MiOiJodHRwczpcL1wvbG9jYWxob3N0Ojk0NDNcL29hdXRoMlwvdG9rZW4iLCJtb2JpbGUiOiI5NDcxMjE4NDUxOCIs" +
+            "Imxhc3RuYW1lRU4iOiJLYWx1Ym93aWxhIiwiZnVsbG5hbWVFTiI6IkRpbXV3YW5DIEthbHVib3dpbGEiLCJzaWQiOiIxYzMwMTBlOS1iYT" +
+            "Q2LTQ1NGMtODVkMS03N2U3ZmIzZTk4N2QiLCJhdWQiOiJ0WFU0RXlsaTRmb0U1SWVuQlZjRGVnTk9mUDBhIiwiY19oYXNoIjoiMXZqRnl" +
+            "FR1FicmdFWnoyM0x3SWVJdyIsIm5iZiI6MTY1NzIwNTk0MiwiYXpwIjoidFhVNEV5bGk0Zm9FNUllbkJWY0RlZ05PZlAwYSIsInVzZXJUe" +
+            "XBlIjoiU09QMSIsImV4cCI6MTY1NzIwOTU0MiwiaWF0IjoxNjU3MjA1OTQyLCJNdWx0aUF0dHJpYnV0ZVNlcGFyYXRvciI6W10sImVtYWl" +
+            "sIjoiZGNrYWx1Ym93aWxhMjUxMzJAZ21haWwuY29tIn0.VM5fxgtpl07oPxd4RNEwW4KvD0rJJV7Lrl9xSIseR2qzoIV3LzZok7Tr9XlLZ" +
+            "mx35MRDkQ7ezar8zZtfTXqaOEAeMIbD0a9yMx14NRhjzPA6iBtdCAdKGaua0RVPZDcr7IUAUsJSJ4Q6S4gYWiOfMHXvrB8zqqx9_rWQTk9" +
+            "Bxn_LkvgU0tyPu88b8G2fUISFTxXb69rJ1mtTfWH_oFGkwKH9ij4NjHp1P2n09S9fVZNT3v1nsY8B5d1bS5YBrRcgtBtPKhNlsQ_4OFq--" +
+            "PMB_OlzZrId6Po9IBarzUrpqY5uzCJGzSk5xAXlxq0jF42xUtAbD5L8CtoESxGMPYRZfg";
+
+    @DataProvider(name = "seperator")
+    public Object[][] getSeperator() {
+
+        return new String[][]{
+                {","},
+                {",,,"}
+        };
+    }
+
+    @DataProvider(name = "requestDataHandler")
+    public Object[][] getRequestStatus() {
+
+        return new String[][]{
+                // When all parameters not null.
+                {"code", "active,OIDC", "BASIC", "Error Login.", "true", "active", "Invalid can handle response for" +
+                        " the request.", "Invalid context identifier."},
+                // When grant_type and login_type are null.
+                {null, "active,OIDC", null, "Error Login.", "true", "active", "Invalid can handle response for the " +
+                        "request.", "Invalid context identifier."},
+                // When all parameters null.
+                {null, null, null, null, "false", null, "Invalid can handle response for the request.", "Invalid " +
+                        "context identifier."}
+        };
+    }
+
+    @DataProvider(name = "commonAuthParamProvider")
+    public Object[][] getCommonAuthParams() {
+
+        return new String[][]{
```

Comment:
```suggestion
        return new String[][] {
```
Check other places for common formatting errors